pkg/adaptation/adaptation.go

```diff
@@ -346,15 +346,40 @@ func (r *Adaptation) startPlugins() (retErr error) {
 		}
 
 		if err := p.start(r.name, r.version); err != nil {
-			return err
+			return fmt.Errorf("failed to start NRI Plugin %q: %w", name, err)
```

Comment:
let's be more explicit in the logs for this start loop... by adding pre-installed... e.g. "starting pre-installed plugins..." at the beginning of this func..

Reply:
to be consistent with the sync changes below these new and start errors should also "continue"

"failed to initialize pre-installed NRI plugin %q: %w"
"failed to start pre-installed NRI plugin %q: %w"